src/mixins/document-mixin.ts

```diff
@@ -29,9 +29,13 @@ export default class DocumentMixin extends Vue {
       }
     }
 
-    let response = await axios.put(
-      url, file, options
-    )
-    return response
+    try {
+      let response = await axios.put(
+        url, file, options
+      )
+      return response
+    } catch (error) {
```

Comment:
A cleaner implementation is to use `.catch()` chained to the put... example on line 17 above.

Reply:
updated